app/src/ui/branches/no-branches.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import { Button } from '../lib/button'
+
+const BlankSlateImage = `file:///${__dirname}/static/empty-no-branches.svg`
+
+interface INoBranchesProps {
+  readonly onCreateNewBranch: () => void
+}
+
+export class NoBranches extends React.Component<INoBranchesProps> {
+  public render() {
+    const shortcut = __DARWIN__ ? '⌘' : 'Ctrl'
+    return (
+      <div className="no-branches">
+        <img src={BlankSlateImage} className="blankslate-image" />
+
+        <div className="title">Sorry, I can't find that branch</div>
+
+        <div className="subtitle">
+          Do you want to create a new branch instead?
+        </div>
+
+        <Button
+          className="create-branch-button"
+          onClick={this.props.onCreateNewBranch}
+          type="submit"
+        >
+          {__DARWIN__ ? 'Create New Branch' : 'Create new branch'}
+        </Button>
+
+        <div className="protip">
+          ProTip! Press <kbd>{shortcut}</kbd> + <kbd>⇧</kbd> + <kbd>N</kbd> to
```

Comment:
The arrow here needs to be `Shift` here on Windows to match up with how menu items draw their shortcuts:

<img width="278" src="https://user-images.githubusercontent.com/359239/31590366-936cb2a2-b25a-11e7-8463-76001ad999a9.png">